packages/genesys-spark-components/src/components/stable/gux-rating/tests/gux-rating.e2e.ts

```diff
@@ -294,15 +294,15 @@ describe('gux-rating', () => {
             },
             {
               press: 'ArrowRight',
-              expectedStarCounts: { emptyStars: 3, fullStars: 2, halfStars: 0 }
+              expectedStarCounts: { emptyStars: 2, fullStars: 3, halfStars: 0 }
```

Comment:
@conor-darcy these test changes are related to your right and left arrow changes in `gux-rating`